pages/docs/tutorials/native/interop-with-c.md

```diff
@@ -41,7 +41,7 @@ linkerOpts.linux = -L/usr/lib/x86_64-linux-gnu -lcurl
 
 A few things are going on in that file so let's see them one by one. The first entry is `headers` which is the list of header files that we want to generate 
 Kotlin stubs for. We can add multiple files to this entry, separating each one with `\` on a new line. In our case we only want `curl.h`. The files we reference
-need to be relative to the folder where the definition file is, or be available on the system path (in our case it would be `/usr/bin/curl`).
+need to be relative to the folder where the definition file is, or be available on the system path (in our case it would be `/usr/include/curl`).
```

Comment:
> The files we reference need to be relative to the folder where the definition file is

I'm not sure it works as described.

> or be available on the system path 

Or on the user-specified path, e.g. with `-I` compiler options.

Reply:
Moreover, `cinterop` doesn't use headers from system paths like `/usr/include` by default.
On macOS it uses Xcode-provided SDKs, on other platforms Kotlin/Native has its own "system roots" (aka sysroots), located somewhere in `~/.konan/dependencies/`.